modules/demo/rapids-api-server/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "rapids-api-server",
+  "version": "0.0.1",
+  "description": "A fastify-based web server that provides browser-access to GPU resources.",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "tap \"test/**/*.test.js\"",
+    "build": "tsc -p ./tsconfig.json",
+    "start": "npm run build; fastify start -l info -P -w app.js",
+    "dev": "fastify start -w -l info -P app.js"
+  },
+  "keywords": ["rapids.ai", "node", "NVIDIA", "gpu", "Dataframe", "pandas"],
+  "author": "Thomson Comer",
+  "license": "MIT",
+  "dependencies": {
+    "@fastify/autoload": "^4.0.0",
+    "@fastify/sensible": "^4.0.0",
+    "@types/node": "17.0.33",
+    "fastify": "^3.0.0",
+    "fastify-arrow": "0.1.0",
```

Comment:
`fastify-arrow@v1.0.0` is the one that supports Arrow v8.0.0

Reply:
Thanks, that'll save me some time for sure!